src/components/AppBar/NavBar.tsx

```diff
@@ -0,0 +1,82 @@
+import React, { useState } from 'react';
+import { Link, useLocation } from 'react-router-dom';
+import Logo from 'assets/images/logo';
+import Hidden from '@material-ui/core/Hidden';
+import * as Style from './NavBar.style';
+import { DonateButtonWithoutFade, DonateButtonWithFade } from './DonateButton';
+import MobileMenu from './MobileMenu';
+import { StyledMobileMenu } from './AppBar.style';
+import MenuIcon from '@material-ui/icons/Menu';
+import CloseIcon from '@material-ui/icons/Close';
+import IconButton from '@material-ui/core/IconButton';
+import ArrowBack from '@material-ui/icons/ArrowBack';
+
+const isLocationPage = (pathname: string) => pathname.includes('/us');
+
+const isHomeOrLocationPage = (pathname: string) =>
+  ['/', '/alert_signup', '/compare'].includes(pathname) ||
+  isLocationPage(pathname);
+
+const NavBar: React.FC = () => {
+  const [isMenuOpen, setMenuOpen] = useState(false);
+  const closeMenu = () => setMenuOpen(false);
+
+  // We only fade the donate button on the home page on mobile, where the donate
+  // button doesn't appear until the banner is scrolled away.
+  const { pathname } = useLocation();
+  const MobileDonateButton = isHomeOrLocationPage(pathname)
+    ? DonateButtonWithFade
+    : DonateButtonWithoutFade;
+
+  return (
+    <Style.AppBar position="sticky" color="transparent" elevation={0}>
+      <Style.Toolbar>
+        <Style.BackLink to="/">
+          {isLocationPage(pathname) && <ArrowBack />}
+        </Style.BackLink>
+        <Link to="/">
+          <Logo />
+        </Link>
+        <Style.Spacer />
+        <Hidden smDown>
+          <Style.NavLink
+            to="/"
+            key="map"
+            activeClassName="active"
+            isActive={(match, { pathname }) => isHomeOrLocationPage(pathname)}
```

Comment:
i dont think this is meant to remain 'active' (underlined, etc.) when on a location page. current behavior on prod is that Map is active/underlined only when on the homepage, and nothing in the navbar is active/underlined when on a location page